### ISSUE TITLE
Restore Travis integration

### ISSUE THREAD

**Issue by mostroverkhov** [rsocket/rsocket-kotlin#17]

**Comment by mostroverkhov** [comment]
@robertroeser Travis fails due to wrong Artifactory credentials. I am somewhat blocked at this issue as there is another library which depends on `RSocket-Android` - hard to distribute it while this is not solved.

**Comment by robertroeser** [comment]
i'll take a look again today

**Comment by robertroeser** [comment]
@mostroverkhov looks like the build based and uploaded to bintray

**Comment by mostroverkhov** [comment]
@robertroeser many thanks! Just checked it - works like a charm

**Comment by mostroverkhov** [comment]
Resolved in #22

**Comment by robertroeser** [comment]
@mostroverkhov its only snapshot builds right now - i need to figure fix releases still...

**Comment by robertroeser** [comment]
@mostroverkhov tried to fix this again a week or so ago...I need to try running publish on my desktop first I think - there seems to be something wrong with it. Right now I'm stumped, because it's using the same setup as RSocket Java.

**Comment by mostroverkhov** [comment]
@robertroeser We started to use Jitpack for distribution - simple tool which can release Github tag/commit with one button click, so I am happy as long as Travis can build snapshots. I think publishing problem can be postponed, unless you are blocked on It.
